django/api/viewsets/application_form.py

```diff
@@ -1,18 +1,34 @@
 from rest_framework.viewsets import ModelViewSet
-from api.serializers.application_form import ApplicationFormSerializer, \
-    ApplicationFormCreateSerializer
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework import status
+
+from api.serializers.application_form import (
+    ApplicationFormSerializer,
+    ApplicationFormCreateSerializer,
+)
 from api.models.go_electric_rebate_application import GoElectricRebateApplication
+from api.models.household_member import HouseholdMember
+from api.serializers.household_member import HouseholdMemberApplicationGetSerializer
 
 
 class ApplicationFormViewset(ModelViewSet):
     queryset = GoElectricRebateApplication.objects.all()
     serializer_classes = {
-        'default': ApplicationFormSerializer,
-        'create': ApplicationFormCreateSerializer,
+        "default": ApplicationFormSerializer,
+        "create": ApplicationFormCreateSerializer,
     }
 
+    @action(detail=True, methods=["GET"], url_path="household")
+    def household(self, request, pk=None):
+        household_member = HouseholdMember.objects.filter(application=pk)
+        serializer = HouseholdMemberApplicationGetSerializer(
+            household_member, many=True
```

Comment:
does this still need the many option?

Reply:
it definitely fails without this AttributeError: 'QuerySet' object has no attribute 'application'